build/configs/rtl8721csm/hello/Make.defs

```diff
@@ -210,10 +211,10 @@ define MAKE_BOARD_SPECIFIC_BIN
   $(TOPDIR)/../build/configs/rtl8721csm/rtl8721csm_make_bin.sh
 endef
 
-KERNEL_BINARY_PATH='$(TOPDIR)/../build/output/bin/km0_km4_image2.bin'
 SECURE_HEADER_SIZE=32
 FLASH_BASE_ADDRESS=0x8000000
 FLASH_SIZE=8388608
+KERNEL_BINARY_PATH='$(TOPDIR)/../build/output/bin/${KERNEL}_${CONFIG_BOARD_BUILD_DATE}.bin'
```

Comment:
Is it possible to use the result of `extract_binname`?

Reply:
No, we cannot use from `extract_binname`.
Because `extract_binname` is working based on the result of build.
But `Make.def` is used to build.

Reply:
How about moving this into define MAKE_SAMSUNG_HEADER?
Then this will be set after build.